.circleci/config.yml

```diff
@@ -64,6 +64,21 @@ jobs:
             CGO_ENABLED=0 equinox release --version=${CIRCLE_SHA1:0:8} --platforms="darwin_amd64 linux_amd64 darwin_386 linux_386 windows_amd64 windows_386" --signing-key=equinox.key --app=$EQUINOX_APP_ID --token=$EQUINOX_ACC_TOKEN --channel dev -- -ldflags "-X main.releaseChannel=dev -X main.build=${CIRCLE_SHA1:0:8}" github.com/rainforestapp/rainforest-cli
             rm -f equinox.key
 
+  release:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Release via GoReleaser
+          command: |
+            export GITHUB_TOKEN=$GITHUB_ACCESS_TOKEN
```

Comment:
We can't configure the name of the env var holding the GitHub token 😞 This line can be removed once the [project settings](https://app.circleci.com/settings/project/github/rainforestapp/rainforest-cli/environment-variables) are updated.